### PR TITLE
bunch of edits

### PR DESCRIPTION
**Pull request by Luci-d**
Removed sixteen 404 links
Added:
email hunter - very best email finder 
spyse - technical search engine 
findsubdomains - service for finding subdomains 
hope it will be helpful

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -349,7 +346,6 @@ Use your browser(CTL-F) to search by country code.
 - [JPEGsnoop](https://sourceforge.net/projects/jpegsnoop)
 - [Metapicz](http://metapicz.com/)
 - [Lunapic](http://www194.lunapic.com/editor)
-- [free-ocr.com](free-ocr.com)
```
Comment (Ph055a) [inline README.md]: Not 404

File: README.md [hunk 2]
```diff
@@ -119,7 +119,6 @@ Use your browser(CTL-F) to search by country code.
 - [Genealogy Links](http://www.genealogylinks.net/)
 - [Hey Press](https://www.hey.press/) (Search for Journalists)
 - [Infobel](http://www.infobel.com/en/world)
-- [Infospace White Pages](http://infospace.com/home/white-pages)
```
Comment (Ph055a) [inline README.md]: Not 404!

File: README.md [hunk 3]
```diff
@@ -229,7 +228,6 @@ Use your browser(CTL-F) to search by country code.
 - [Factual Search](https://factualsearch.news)
 
 #### Censorship, Takedowns, Copyright
-- [Google Copyright Takedowns](https://www.google.com/transparencyreport/removals/copyright/explore/?)
```
Comment (Ph055a) [inline README.md]: Not 404, just needs the trailing ? in the url to be removed.

File: README.md [hunk 4]
```diff
@@ -554,6 +548,7 @@ Use your browser(CTL-F) to search by country code.
 - [TRExpertWitness](https://trexpertwitness.com/)
 
 #### E-mail Search / Check
+- [Email Hunter](https://hunter.io/)
```
Comment (Ph055a) [inline README.md]: Data is not actionable or free, I cannot accept this.

File: README.md [hunk 5]
```diff
@@ -842,7 +834,6 @@ Use your browser(CTL-F) to search by country code.
 - [Publicwww](https://publicwww.com/)
 
 #### General Search
-- [isearchfrom](isearchfrom.com)
```
Comment (Ph055a) [inline README.md]: Not 404

File: README.md [hunk 6]
```diff
@@ -902,7 +894,6 @@ Use your browser(CTL-F) to search by country code.
 - [Swiss Cows](https://swisscows.com/)
 - [bvsg.org](http://bvsg.org)
 - [webcitation](http://www.webcitation.org)
-- [AWS Buckets - Gray Hat Warfare](buckets.grayhatwarfare.com)
```
Comment (Ph055a) [inline README.md]: Not 404

File: README.md [hunk 7]
```diff
@@ -1078,5 +1066,4 @@ Use your browser(CTL-F) to search by country code.
 - [Melissa Data](http://www.melissadata.com/lookups/index.htm)
 
 #### Other Interesting
-- [iknowwhatyoudownload](https://iknowwhatyoudownload.com/en/)
-- [Amazon Wish List Search](www.amazon.com/gp/registry/search)
+- [Findsubdomains](https://findsubdomains.com/)
```
Comment (Ph055a) [inline README.md]: Can't accept this, I already have it in another repo https://github.com/Ph055a/Domains_OSINT/blob/master/README.md

File: README.md [hunk 8]
```diff
@@ -990,8 +980,6 @@ Use your browser(CTL-F) to search by country code.
 - [Airport Webcams](http://airportwebcams.net/)
 - [Live ATC](https://www.liveatc.net/)
 - [Radar Box](https://www.radarbox24.com)
-- [Radar Zhaw](http://radar.zhaw.ch/AirTraffic.html)
-- [http://webtrak5.bksv.com]()
```
Comment (Ph055a) [inline README.md]: Not 404, just missing description

**Comment by Luci-d** [comment]
> Many of these are not 404's and I will accept spyse.com but not the others.
> 
> Thank You

Oh, I'm sure I checked at list five of them manually. But it's cool. Thank you

**Pull request merged**